src/test/resources/pages/adminEmailCompose.html

```diff
@@ -27,7 +27,7 @@
          </script>
          <script src="/tinymce/js/tinymce/tinymce.min.js" type="text/javascript">
          </script>
-         <link href="http://localhost:8888/tinymce/js/tinymce/skins/lightgray/skin.min.css" rel="stylesheet"/>
```

Comment:
Any possible reason why the localhost is not replaced? I run God mode as mentioned before and it got replaced by `${app.url}` as expected.